CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
 # Change Log
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
-## 0.2.0 - 2017-11-02
+## 0.2.0 - 2017-11-10
 ### Added
-DocumentDB and MongoDB accounts now support:
-- View/Create/Delete databses, collections, and documents
-- Click on a document to open in the editor
-- Edit a document and persist changes to the cloud
-- 'Load more' documents in the explorer
+- Graph:
+  - View/Create/Delete databases and graphs
+  - Click on a graph to open graph viewer
```

Comment:
How about 

- Click on a graph to visualize data
- Query graph using gremlin (see https://docs.microsoft.com/en-us/azure/cosmos-db/gremlin-support)